internal/gui/gui_unix.go

```diff
@@ -3,8 +3,23 @@
 
 package gui
 
-import "github.com/skycoin/skywire/pkg/util/osutil"
+import (
+	"os/user"
+
+	"github.com/skycoin/skywire/pkg/util/osutil"
+)
 
 func platformExecUninstall() error {
 	return osutil.Run("/bin/bash", "-c", deinstallerPath)
 }
+
+func checkRoot() bool {
+	thisUser, err := user.Current()
```

Comment:
Same suggestion as above. Using `this` sounds odd to me.

Reply:
done